frontend/src/store.ts

```diff
@@ -10,7 +10,9 @@ declare global {
   }
 }
 
-const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+  // eslint-disable-next-line no-mixed-operators
```

Comment:
required?

Reply:
Makes life easier as it enables traces in redux-devtools (you can see where an action was triggered from)